File: frontend/src/schemas/movie.ts
class Movie {
    id:number
    titulo: string;
    año: number;
    url: string;
    description: string;

    constructor(id:number,titulo: string, año: number, url: string, description: string) {
        this.id = id
        this.titulo = titulo;
        this.año = año;
        this.url = url;
        this.description = description;
    }
}

export {Movie}

File: backend/modelo/models/movie.py
from modelo.config.database import Base
from sqlalchemy import Column,Integer,String

class Movie(Base):
    __tablename__ = "Movies"
    
    id = Column(Integer,primary_key=True)
    titulo = Column(String)    
    año = Column(Integer)
    url = Column(String)
    description = Column(String)
    